src/controllers/user.controller.ts

```diff
@@ -0,0 +1,74 @@
+import User from '../models/user.model'
+import { Request, Response } from 'express'
+
+export const UserController = {
+  async getAllUsers(req: Request, res: Response) {
```

Comment:
For the controllers, let's avoid putting them in classes. Let's use classes only for services, for controllers we shall export them as individual functions like this. Do this to all functions in all of your controller files. 
```suggestion
  export const getAllUsers = async (req: Request, res: Response) => {
```